src/assets/ethereum-erc20-tokens.json

```diff
@@ -1,66 +1,82 @@
     "decimals": 18,
+    "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/b69af6c4c7b5734900824a6571a9ae3bbf59ec54/blockchains/polygon/info/logo.png"
+  },
+  {
+    "name": "Dai Stablecoin",
+    "address": "0xD7E55eB808693D5Ff81a3391c59886C7E0449f35",
     "chainId": 5,
-    "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png"
+    "symbol": "DAI",
+    "decimals": 18,
+    "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/logo.png"
   },
   {
     "name": "Wrapped Ether",
     "address": "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
+    "chainId": 5,
     "symbol": "WETH",
     "decimals": 18,
-    "chainId": 5,
     "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png"
   },
   {
-    "name": "Dai Stablecoin",
-    "address": "0xD7E55eB808693D5Ff81a3391c59886C7E0449f35",
-    "symbol": "DAI",
-    "decimals": 18,
+    "name": "Uniswap",
+    "address": "0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984",
     "chainId": 5,
-    "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/logo.png"
+    "symbol": "UNI",
+    "decimals": 18,
+    "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png"
   },
   {
-    "name": "Matic Token",
-    "address": "0x4701Aa9471d7bfAc765D87dcb1Ea6BB23AD32733",
-    "chainId": 5,
+    "name": "Polygon",
+    "address": "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0",
+    "chainId": 1,
     "symbol": "MATIC",
     "decimals": 18,
     "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/b69af6c4c7b5734900824a6571a9ae3bbf59ec54/blockchains/polygon/info/logo.png"
   },
   {
     "name": "Wrapped Ether",
     "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    "chainId": 1,
     "symbol": "WETH",
     "decimals": 18,
-    "chainId": 1,
     "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png"
   },
   {
-    "name": "Dai Stablecoin",
-    "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
-    "symbol": "DAI",
-    "decimals": 18,
+    "name": "Wrapped Bitcoin",
+    "address": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
     "chainId": 1,
-    "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/logo.png"
+    "symbol": "WBTC",
+    "decimals": 8,
+    "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599/logo.png"
   },
   {
-    "address": "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0",
-    "name": "Polygon",
-    "symbol": "MATIC",
-    "decimals": 18,
+    "name": "Tether",
```

Comment:
This should be named `Tether USD`, which is the [official name](https://etherscan.io/address/0xdAC17F958D2ee523a2206206994597C13D831ec7#readContract#F1)